base_custom_info/views/custom_info_value_view.xml

```diff
@@ -1,19 +1,53 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
+<!-- Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). -->
+<odoo>
 
 <record id="base_custom_info_value_tree" model="ir.ui.view">
     <field name="name">base.custom.info.value.tree</field>
     <field name="model">custom.info.value</field>
     <field name="arch" type="xml">
         <tree string="Custom Property Values">
```

Comment:
I don't think so. This will lead to conflicts (lines that doesn't belong to the same model), and it also break the concept of information template. If you need something extra, make another profile. The only problem I see with this that the same property can't be reused for several templates. Or it can be?